frontend/src/main.tsx

```
@@ -3,11 +3,14 @@ import { createRoot } from 'react-dom/client'
 import { BrowserRouter } from 'react-router-dom';
 import './index.css'
 import App from './App.tsx'
+import { DisasterProvider } from './providers/DisasterContextProvider';
 
 createRoot(document.getElementById('root')!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)